docs/modular-resources/key-concepts.md

```diff
@@ -104,29 +104,32 @@ If you are [creating a custom module](/modular-resources/create/) and [uploading
 - Your model triplet must be all-lowercase.
 - Your model triplet may only use alphanumeric (`a-z` and `0-9`), hyphen (`-`), and underscore (`_`) characters.
 
-In addition, you should chose a name for the `family` of your model based on the whether your module implements a single model, or multiple models:
+For the middle segment of your model triplet `repo-name`, use the name of the git repository where you store your module's code.
+The `repo-name` should describe the common functionality provided across the model or models of that module.
 
-- If your module provides a single model, the `family` should match the `subtype` of whichever API your model implements.
-  For example, the Intel Realsense module `realsense`, available from the [Viam registry](https://app.viam.com/module/viam/realsense), implements the `camera` component API, so it is named as follows:
+For example:
+
+- The `rand:yahboom:arm` model and the `rand:yahboom:gripper` model uses the repository name [yahboom](https://github.com/viam-labs/yahboom).
+  The models implement the `rdk:component:arm` and the `rdk:component:gripper` API to support the yahboom dofbot arm and gripper respectively.
 
   ```json {class="line-numbers linkable-line-numbers"}
   {
-    "api": "rdk:component:camera",
-    "model": "viam:camera:realsense"
+    "api": "rdk:component:arm",
+    "model": "rand:yahboom:arm"
+  },
+  {
+    "api": "rdk:component:arm",
+    "model": "rand:yahboom:arm"
   }
   ```
 
-- If your module provides multiple models, the `family` should describe the common functionality provided across all the models of that module.
-  For example, the ODrive module `odrive`, available from the [Viam registry](https://app.viam.com/module/viam/odrive), implements several `motor` component APIs, so it is named as follows:
+- The `viam-labs:audioout:pygame` model uses the repository name [audioout](https://github.com/viam-labs/audioout)
+  It implements the custom API `viam-labs:service:audioout`.
 
   ```json {class="line-numbers linkable-line-numbers"}
```

Comment:
Same here; I think this should say something like "The following example JSON shows how this model would be configured in the robot's component config:" (or whatever is actually accurate; I don't think my words are)